nubis/bin/packer-bootstrap

```diff
@@ -53,6 +53,13 @@ elif [[ -f /etc/os-release ]]; then
    . /etc/os-release
 
    if [[ "$NAME" = "Amazon Linux AMI" ]]; then
+
+      # Disable auto-updating!
+      echo "Disabling Amazon Auto-Updating"
+      sudo perl -pi -e's[^(\s*)repo_upgrade:.*][$1repo_upgrade: false]g' /etc/cloud/cloud.cfg
+      sudo /usr/sbin/update-motd --disable
```

Comment:
This is a naughty way to get what I want. Better would be to disable the yum plugin for the duration of our provisionning and re-enable it at the end.